crates/compiler/fmt/src/annotation.rs

```diff
@@ -498,6 +498,109 @@ fn format_assigned_field_help<T>(
     }
 }
 
+impl<'a> Formattable for RecordBuilderField<'a> {
+    fn is_multiline(&self) -> bool {
+        is_multiline_record_builder_field_help(self)
+    }
+
+    fn format_with_options(&self, buf: &mut Buf, _parens: Parens, newlines: Newlines, indent: u16) {
+        // we abuse the `Newlines` type to decide between multiline or single-line layout
```

Comment:
Can you elaborate what the abuse is? I don't think it's an abuse, I think it's what it's meant for!

Reply:
I got this comment from copying the [`Formattable` impl for `AssignedField`](https://github.com/agu-z/roc/blob/f94004d1372044e61f0f960e896a2853462080c5/crates/compiler/fmt/src/annotation.rs#L404), but I also didn't think this was abusing it